packages/web-twig/src/Resources/components/Item/Item.twig

```diff
@@ -0,0 +1,70 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _elementType = props.elementType | default('button') -%}
+{%- set _label = props.label | default('') -%}
+{%- set _unsafeLabel = props.UNSAFE_label | default(null) -%}
+{%- set _helperText = props.helperText | default(null) -%}
+{%- set _iconName = props.iconName | default(null) -%}
+{%- set _isDisabled = props.isDisabled | default(null) -%}
+{%- set _isSelected = props.isSelected | default(null) -%}
+{%- set _unsafeHelperText = props.UNSAFE_helperText | default(null) -%}
+{%- set _href = props.href | default(null) -%}
+{%- set _target = props.target | default(null) -%}
+{%- set _type = props.type | default(null) -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'Item' -%}
+{%- set _disabledClassName = _isDisabled ? _spiritClassPrefix ~ 'Item--disabled' : null -%}
+{%- set _selectedClassName = _isSelected ? _spiritClassPrefix ~ 'Item--selected' : null -%}
+{%- set _iconClassName = _spiritClassPrefix ~ 'Item__icon' -%}
+{%- set _startIconClassName = _iconName ? _iconClassName ~ ' ' ~ _spiritClassPrefix ~ 'Item__icon--start'  -%}
```

Comment:
Do we need these conditions? We can set the className props every time, can't we? 

We can also use classProp with an array, but this works too.

Reply:
conditions removed, they are in the component, so we don't need them here